LEAF_Request_Portal/admin/ajaxIndex.php

```diff
@@ -53,6 +53,7 @@ function checkToken()
 
         break;
     case 'remove_user_old':
+        // this should be deprecated as of 8/18/2023
```

Comment:
Need to track deprecations via GitHub Issues. Example: https://github.com/department-of-veterans-affairs/LEAF/issues/2062